repos/tap/components/sidebar/content/fileTreePanel.js

```diff
@@ -49,7 +49,7 @@ const FileTreeHeader = ({styles, title}) => {
     </View>
   )
 }
-
+const treeScroll = {overflowY: "scroll", height: "100vh", paddingBottom: "9em" }
```

Comment:
This should not be defined inline like this.
Either move it to a theme file, or use `reStyle` from retheme